crystalfontz/dbus/service.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+import logging
+import os
+from typing import Optional
+
+import click
+from sdbus import (  # pyright: ignore [reportMissingModuleSource]
```

Comment:
I'm getting some warnings about missing module source. I think this is working, but there are other spots where I'm missing this annotation.